frontend/server/libs/Validators.php

```diff
@@ -102,12 +102,20 @@ public static function validateValidAlias(
         if (!is_string($parameter) ||
             empty($parameter) ||
             strlen($parameter) > 32 ||
-            preg_match('/^[a-zA-Z0-9-_]+$/', $parameter) !== 1
+            !self::isValidAlias($parameter)
         ) {
             throw new InvalidParameterException('parameterInvalidAlias', $parameterName);
         }
     }
 
+    /**
+     * @param string $parameter
+     * @return boolean
+     */
+    public static function isValidAlias($parameter) : bool {
```

Comment:
```suggestion
    public static function isValidAlias(string $parameter) : bool {
```

argh, se me fue esta omisión de tipo.